ui/apps/app-center/src/extensions/install-modal.tsx

```diff
@@ -84,9 +84,8 @@ const IntegrationInstallModal: React.FC<RootExtensionProps> = props => {
       dismissLabel={t('Dismiss')}
       modalTitleLabel={t('App Install')}
       integrationName={integrationName}
-      installTitleLabel1={t('To add')}
-      installTitleLabel2={t(
-        'to your World we have to do a few things first. This will take less than a minute.',
+      installTitleLabel={t(
+        'To add {{integrationName}} to your World we have to do a few things first. This will take less than a minute.',
```

Comment:
`integrationName` should be passed to the second param of tFunc.
`t('{{variableName}}', { variableName: 'my var name' })`

Reply:
ok, I changed these now